### ISSUE TITLE
[Gazebo] Update test name to test string in the table UI

### ISSUE THREAD

**Issue by RulaKhaled** [codecov/engineering-team#2432]
design: https://www.figma.com/design/TcppABudxnslgEmaXm0B9O/GH-2028?node-id=1-2&t=1c8whZB94GIederY-0

**Comment by ajay-sentry** [comment]
@Adal3n3 @RulaKhaled  Do you remember what this ticket was referencing?

**Comment by Adal3n3** [comment]
@ajay-sentry yes, I remember we used to have test name only in the table but we want to include the path - the copy-pastable test name (test string) in the table. Can you check if we support it in all languages?

**Comment by ajay-sentry** [comment]
@Adal3n3 Not all languages I don't think, I believe it's a configurable option in vitest, and jest, and available for free in pytest and php? 

@joseph-sentry would know for sure, but its def not for all languages. I think for all languages would be really hard to do because it'd require understanding how all those frameworks pass in file/test name and parse them accordingly
